portfolio/src/main/webapp/script.js

```diff
@@ -91,6 +91,14 @@ function close_comments() {
   document.querySelector('.comments_modal').style.display = 'none';
 }
 
+function open_translate() {
```

Comment:
Personally I would separate this file by module: put all the translate functions together, put all the comment functions together, etc.

If the file becomes too large, also consider refactor some modules out as their own js file.

Reply:
Sounds good thank you Frank!